server/asset/eth/rpcclient_harness_test.go

```diff
@@ -153,9 +153,18 @@ func testAccountBalance(t *testing.T, assetID uint32) {
 		t.Fatalf("accountBalance error: %v", err)
 	}
 
-	diff := new(big.Int).Sub(balBefore, balAfter)
-	if diff.Cmp(dexeth.GweiToWei(vGwei)) <= 0 {
-		t.Fatalf("account balance changed by %d. expected > %d", dexeth.WeiToGwei(diff), uint64(vGwei))
+	if assetID == BipID {
+		diff := new(big.Int).Sub(balBefore, balAfter)
+		if diff.Cmp(dexeth.GweiToWei(vGwei)) <= 0 {
+			t.Fatalf("account balance changed by %d. expected > %d", dexeth.WeiToGwei(diff), uint64(vGwei))
+		}
+	}
+
+	if assetID == testTokenID {
```

Comment:
Will you beef up the commit message with notes about these other fixes, and changes like using delta, etc.?